subsys/bluetooth/controller/ll_sw/ull_master.c

```diff
@@ -555,6 +587,57 @@ uint8_t ll_enc_req_send(uint16_t handle, uint8_t const *const rand,
 }
 #endif /* CONFIG_BT_CTLR_LE_ENC */
 
+void ull_master_cleanup(struct node_rx_hdr *rx_free)
+{
+	struct node_rx_ftr *ftr = &rx_free->rx_ftr;
+	struct ll_scan_set *scan =
+		(void *)HDR_LLL2EVT(ftr->param);
```

Comment:
This could be one line

Reply:
If this is not too much of a concern, I will stick to 80-column just to avoid github side-by-side compare from not to line wrap.

Reply:
Oh, I guess that might be an issue on smaller screens :) It's a minor comment that doesn't need to be fixed; however you are likely going to see future commits with 100 length lines anyhow ^_^